custom_model_runner/datarobot_drum/resource/predict_mixin.py

```diff
@@ -118,6 +140,9 @@ def _transform(self, logger=None):
             feature_binary_data, feature_mimetype, feature_charset = self._fetch_data_from_request(
                 "X", logger=logger
             )
+            mimetype_support_error_response = self._check_mimetype_support(feature_mimetype)
```

Comment:
@4dahalibut  this PR affects custom transform code. please have a look

Reply:
so for transforms we want pyarrow to be the only option...

Reply:
right now it can be either Arrow or CSV, correct?
are you going to drop CSV support?

Reply:
cc @yakov-g

Reply:
Yes that is what I would like to do @jlpolit

Reply:
if everyone's on board with that, I'll make that change

Reply:
let's hear from @yakov-g - Arrow complicates DRUM installation

Reply:
So....with current approach, if `pyarrow` won't be installed `transform`; user will see the error `install pyarrow` - looks  ok to me.

We could also add an early check for pyarrow here:
- https://github.com/datarobot/datarobot-user-models/blob/1bff8dff4c0d21e23d2196f727a5eb68cee1259f/custom_model_runner/datarobot_drum/drum/args_parser.py#L832
- Something like this:
```
if options.target_type == "transform":
   try:
       import pyarrow
    except ImportError:
       print("transform requires arrow to be supported")
        exit(1)
```

Reply:
why can't we just install pyarrow

Reply:
if installing pyarrow is not terribly bad - simly installing it would make things simpler for everyone and easier to maintain and test

Reply:
We can require pyarrow - only issues is that some Py and arrow versions are not cross compatible.

Reply:
Btw, even if we keep `pyarrow` in our requirements file, this PR should go in as is.

Reply:
do you mean so that in case `pyarrow` installation fails on some python version or platform - we'll still have fully functional DRUM just without Arrow?
oh, and without custom transforms (if Arrow is the only format there)

Reply:
well nonfunctional combinations will probably only be found in developer setups... So I say we proceed with that for now.

Reply:
> do you mean so that in case `pyarrow` installation fails on some python version or platform - we'll still have fully functional DRUM just without Arrow?
> oh, and without custom transforms (if Arrow is the only format there)

Yes. Also people could install with `--no-deps`